appengine-managed-runtime/src/test/java/com/google/apphosting/utils/config/AppEngineWebXmlProcessorTest.java

```diff
@@ -0,0 +1,2076 @@
+    assertEquals(null, settings.getHost());
+  }
+
+  public void testHealthCheck_getTimeoutSec_invalid() {
+    String xml =
+        "<appengine-web-app xmlns=\"http://appengine.google.com/ns/1.0\">\n"
+            + "<application>helloworld-java</application>\n"
+            + "<version>1</version>\n"
+            + "<health-check>\n"
+            + " <timeout-sec>123abc</timeout-sec>\n"
+            + "</health-check>\n"
+            + "</appengine-web-app>\n";
+
+    try {
+      parseAngGetHealthCheck(xml);
+      fail();
+    } catch (AppEngineConfigException expected) {
+    }
+  }
+
+  public void testHealthCheck_getTimeoutSec_negative() {
+    String xml =
+        "<appengine-web-app xmlns=\"http://appengine.google.com/ns/1.0\">\n"
+            + "<application>helloworld-java</application>\n"
+            + "<version>1</version>\n"
+            + "<health-check>\n"
+            + " <timeout-sec>-1</timeout-sec>\n"
+            + "</health-check>\n"
+            + "</appengine-web-app>\n";
+
+    try {
+      parseAngGetHealthCheck(xml);
+      fail();
+    } catch (AppEngineConfigException expected) {
+    }
+  }
+
+  public void testHealthCheck_getUnhealthyThreshold() {
+    String xml =
+        "<appengine-web-app xmlns=\"http://appengine.google.com/ns/1.0\">\n"
+            + "<application>helloworld-java</application>\n"
+            + "<version>1</version>\n"
+            + "<health-check>\n"
+            + " <unhealthy-threshold>7</unhealthy-threshold>\n"
+            + "</health-check>\n"
+            + "</appengine-web-app>\n";
+    AppEngineWebXml.HealthCheck settings = parseAngGetHealthCheck(xml);
+    assertFalse(settings.isEmpty());
+    assertEquals(true, settings.getEnableHealthCheck());
+    assertEquals(null, settings.getCheckIntervalSec());
+    assertEquals(null, settings.getTimeoutSec());
+    assertEquals(7, settings.getUnhealthyThreshold().intValue());
+    assertEquals(null, settings.getHealthyThreshold());
+    assertEquals(null, settings.getRestartThreshold());
+    assertEquals(null, settings.getHost());
+  }
+
+  public void testHealthCheck_getUnhealthyThreshold_invalid() {
+    String xml =
+        "<appengine-web-app xmlns=\"http://appengine.google.com/ns/1.0\">\n"
+            + "<application>helloworld-java</application>\n"
+            + "<version>1</version>\n"
+            + "<health-check>\n"
+            + " <unhealthy-threshold>123abc</unhealthy-threshold>\n"
+            + "</health-check>\n"
+            + "</appengine-web-app>\n";
+
+    try {
+      parseAngGetHealthCheck(xml);
+      fail();
+    } catch (AppEngineConfigException expected) {
+    }
+  }
+
+  public void testHealthCheck_getUnhealthyThreshold_negative() {
+    String xml =
+        "<appengine-web-app xmlns=\"http://appengine.google.com/ns/1.0\">\n"
+            + "<application>helloworld-java</application>\n"
+            + "<version>1</version>\n"
+            + "<health-check>\n"
+            
```

Comment:
Fine as is but `assertEquals(true, ..` could be replaced with `assertTrue` and `assertEquals(null, ...` could be replaced with `assertNull`